.classpath

```diff
@@ -24,5 +24,7 @@
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/gcs dependencies"/>
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/google-api-client-appengine-1.18.0-rc.jar"/>
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/google-api-client-1.18.0-rc.jar"/>
+	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/repackaged-appengine-jakarta-jstl-1.1.2.jar"/>
+	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/repackaged-appengine-jakarta-standard-1.1.2.jar"/>
```

Comment:
This seems to be the latest from a quick search, there doesn't seem to be anymore official support for JSTL so it seems to not have been updated for awhile, this same version is still used by some in year 2013 and year 2015 despite it being introduced in 2009 or before.